### ISSUE TITLE
Playlist persistence from Grayjay Youtube plugin 

### ISSUE THREAD

**Issue by Aikarus** [futo-org/grayjay-android#792]
Any playlists made on Grayjay using the Youtube plugin do not persist to Youtube itself.

**Comment by adam-devel** [comment]
The ability to sync local playlists with youtube would be great

**Comment by fogoplayer** [comment]
This would be great. My ideal use case for GrayJay is that, on a per-service level, it's just a client for the service. I don't want a separate, imported copy of my playlists—I want to access my playlists, as they are on YouTube, from the app

**Comment by Zvonimir-FUTO** [comment]
Related to but not same: #857

**Comment by adam-devel** [comment]
> Related to but not same: #857 

the same feature, just in the opposite direction: [Grayjay -> YouTube] instead of [YouTube -> Grayjay]

https://github.com/futo-org/grayjay-android/issues/857#issuecomment-2200623525

**Comment by J-Liebl** [comment]
Yes would be very nice. I use Grayjay on Phone and Youtube to dig music (also because of the algorithm) on my PC. At the moment I have to 2 playlists for each playlist which is really really annoying. This leads to two playlists which should be same, but arent.
